api/CharacterData.json

```diff
@@ -0,0 +1,393 @@
+{
+  "api": {
+    "CharacterData": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CharacterData",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": "6"
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        }
+      },
```

Comment:
Sorry, I just noticed that [status information](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#status-information) are missing, both for the feature and each sub-feature. The information can be derived from the [Specifications section on MDN](https://developer.mozilla.org/de/docs/Web/API/CharacterData#Specification).

Since the [DOM Standard](https://dom.spec.whatwg.org/#characterdata) doesn't mention the `ChildNode` interface, this sub-feature should be probably marked `"standard_track": false" and everything else `"standard_track": true".